### PR TITLE
Use ncurses to notify of broken file instead of printf()

### LINKED ISSUE #6: UI breaks when wavs are corrupted
A message will pop up in the middle of the UI and break everything. This message should be moved somewhere else, or should halt the program.
image

### PR DESCRIPTION
**Pull request by WillNilges**
Closes #6

### CODE REVIEW

File: sampleZone.c [hunk 1]
```diff
@@ -135,6 +139,22 @@ int main(int argc, char *argv[]) {
     int selectedFileIndex = 0;
     audioFile *selectedFile = &files[selectedFileIndex];
 
+    //Check files for errors (TODO: More robust error checking code. Make this not broken)
```
Comment (WillNilges) [inline sampleZone.c]: This is step 2 of the process. My PR will fix the bug so the program doesn't break, and I'm going to improve the feature by properly listing the broken files on load.

**Comment by mfrancis95** [comment]
Check 7fcfc8cd752bf716fbac690f2ff94b430ea8a996. `WINDOW_HEIGHT` was renamed to `DEFAULT_WINDOW_HEIGHT`.

**Comment by WillNilges** [comment]
Oh, okay, thanks.

**Comment by sgreene570** [comment]
@WillNilges  can you rebase and squash?

**Comment by WillNilges** [comment]
Yes.

On Jan 23, 2019 18:42, "Stephen Greene" <notifications@github.com> wrote:

@WillNilges <https://github.com/WillNilges> can you rebase and squash?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/sgreene570/sampleZone/pull/14#issuecomment-457010196>,
or mute the thread

**Pull request merged**